### PR TITLE
SITES-354 Add links between public and editorial views where appropriate

### PR DESCRIPTION
**Pull request by maxious**

### CODE REVIEW

**Comment by ltankey** [comment]
This is a useful function, but should we consider putting it into the user's navbar? Viewing a published page is akin to switching contexts, which would be good to do from the top navbar.

**Comment by micapam** [comment]
Yeah I agree - published view should be clean. However link to 'view published page' is fine to be on the editorial page view itself, IMO...

**Comment by maxious** [comment]
I've moved it up to the top bar... which makes it obvious that you could reduce the number of links in there if you made people go to the first three links before they could access those functions (Editorial Root, This Section Editorial, This page in Editorial). Thoughts?

File: app/assets/stylesheets/page_controls.scss [hunk 1]
```diff
@@ -0,0 +1,43 @@
+@media screen and (min-width: 75em) {
+[class^='controls'] {
+  section.left {
+    width: 75% !important;
```
Comment (coderdan) [inline app/assets/stylesheets/page_controls.scss]: I'm guessing this is a temporary measure until we have the UI kit update?

File: app/views/application/_page_actions.html.haml [hunk 2]
```diff
@@ -1,17 +1,26 @@
-%li
-  = link_to @section.name, editorial_section_path(@section)
-- if show_new_page_link?
-  %li
-    = link_to 'New page', prepare_editorial_section_nodes_path(@section, parent_id: @node)
+%ul.secondary-controls
+  - if node and node.section
```
Comment (coderdan) [inline app/views/application/_page_actions.html.haml]: Prefer && to and

**Comment by coderdan** [comment]
I know Treb is doing some more work on the visual treatment here so for now LGTM

**Comment by micapam** [comment]
LGTM too.

**Pull request merged**